src/drivers/net/bnxt/bnxt.h

```diff
@@ -885,15 +886,19 @@ struct bnxt {
 #define DID_57508_VF_HV      0x1806
 #define DID_57508_VF_RDMA_HV 0x1807
 /* Stratus Device IDs */
-#define DID_57320_1         0x16F0
-#define DID_57320_2         0x16F1
-#define DID_57454_MHB       0x1604
-#define DID_57454_MHB_RDMA  0x1605
-#define DID_57454_VF_RDMA   0x1606
-#define DID_57454_VF        0x1609
-#define DID_57454           0x1614
-#define DID_58802           0xD802
-#define DID_58804           0xD804
+#define DID_57320_1          0x16F0
+#define DID_57320_2          0x16F1
+#define DID_57454_MHB        0x1604
+#define DID_57454_MHB_RDMA   0x1605
+#define DID_57454_VF_RDMA    0x1606
+#define DID_57454_VF         0x1609
+#define DID_57454            0x1614
+#define DID_58802            0xD802
+#define DID_58804            0xD804
+#define DID_57454_VF_HV      0x1607
+#define DID_57454_VF_HV_RDMA 0x1608
+#define DID_57412_VF_HV      0x16c5
+#define DID_57412_VF_HV_RDMA 0x16bd
 
```

Comment:
With the new flag, these defines are unused, correct?

Reply:
Correct observation.  I have removed the unnecessary defines as suggested